Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -0,0 +1,267 @@
+                HStack {
+                    NavigationLink {
+                        valuesView
+                    } label: {
+                        Text("Value")
+                        Spacer()
+                        if let value = model.selectedValue as? CodedValue {
+                            Text(value.name)
+                                .frame(maxWidth: .infinity, alignment: .trailing)
+                        }
+                    }
+                }
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+            } else {
+                HStack {
+                    Text("Value")
+                    Spacer()
+                    if let value = inputValue {
+                        Text(value.formatted())
+                    }
+                }
+                .contentShape(Rectangle())
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+                .onTapGesture {
+                    inputBoxIsPresented = true
+                }
+                .alert("Provide a comparison value", isPresented: $inputBoxIsPresented, actions: {
+                    TextField("10", value: $inputValue, format: .number)
+                        .keyboardType(.numberPad)
+                    Button("Done") {
+                        inputBoxIsPresented = false
+                        model.selectedValue = inputValue
+                    }
+                    Button("Cancel") {
+                        inputBoxIsPresented = false
+                        model.selectedValue = nil
+                        inputValue = nil
+                    }
+                })
+            }
+        }
+        .navigationTitle("Add Condition")
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .confirmationAction) {
+                Button("Done") {
+                    isConditionMenuPresented.toggle()
+                    model.addConditionalExpression()
+                    model.selectedValue = nil
+                    inputValue = nil
+                }
+                .disabled(
+                    model.selectedAttribute == nil ||
+                    model.selectedComparison == nil ||
+                    model.selectedValue == nil
+                )
+            }
+            ToolbarItem(placement: .cancellationAction) {
+                Button("Cancel") {
+                    isConditionMenuPresented.toggle()
+                    model.clearSelection()
+                    inputValue = nil
+                }
+            }
+        }
+        .navigationViewStyle(.stack)
+    }
+    
+    @ViewBuilder var attributesView: some View {
+        List(model.possibleAttributes, id: \.name) { attribute in
+            HStack {
+                Text(attribute.name)
+                Spacer()
+                if attribute === model.selectedAttribute {
```

Comment:
The spacer is only needed if the image is displayed.

```suggestion
                if attribute === model.selectedAttribute {
                    Spacer()
```

Reply:
The current implementation supports tapping anywhere on the list item to select the value which seems like a better user experience. Removing the `Spacer()` causes only the text to be selectable.

Reply:
The tap gesture should still work on the entire row since it is on the H stack and not on just the text.

Reply:
I removed the spacers but still see the same behavior where only the text and not the entire list row can be tapped.